src/page/managers/LoginManager.ts

```diff
@@ -305,6 +336,15 @@ export default class LoginManager {
   static async fetchAndHydrate(onesignalId: string): Promise<void> {
     logMethodCall('LoginManager.fetchAndHydrate', { onesignalId });
 
+    const newRecordsState = OneSignal.coreDirector.getNewRecordsState();
+
+    if (!newRecordsState) {
+      Log.error(`fetchAndHydrate: NewRecordsState is undefined`);
+    } else {
+      const delay = newRecordsState.OP_REPO_POST_CREATE_DELAY;
+      await awaitableTimeout(delay);
```

Comment:
The `delay` variable doesn't add anything here, recommend we just use OP_REPO_POST_CREATE_DELAY directly.
* _The rules of thumb I try to use in combining lines of code or splitting them out is; split up long lines / very nested patterns to access a value, logic getting a value (unless it is very simply), and some subjectiveness; less code is normally better; less bugs and quicker to read (but if it isn't obviously clear to other reads then splitting lines up is better)_